layout.tt

```diff
@@ -36,6 +36,7 @@ BLOCK navigationLink %]
   <link rel="shortcut icon" type="image/png" href="/favicon.png" />
   <meta property="og:image" content="/logo/nixos-lores.png" />
   <meta name="google-site-verification" content="ir-07nYvo3u3x_VmkTO1wCfYJ8uC-SrVBGR7hZgqPSE" />
+  <meta name="description" content="A package manager and operating system providing an unique approach to packaging and system configuration" />
```

Comment:
The description is used when rendering a search result from a search engine

Reply:
@zMitchell We have to live with what we have for now. Rewriting the web site will inevitably be a long incremental process, because Big Bang changes evidently don't fly.

Reply:
That was a suggestion to figure out a similar mechanism, not replace the site generator